code/modules/power/port_gen.dm

```diff
@@ -177,11 +177,9 @@
 	..()
 
 /obj/machinery/port_gen/pacman/proc/process_exhaust()
-	var/decl/material/mat = GET_DECL(sheet_material)
-	if(mat && mat.burn_product)
-		var/datum/gas_mixture/environment = loc.return_air()
-		if(environment)
-			environment.adjust_gas(mat.burn_product, 0.05*power_output)
+	var/decl/material/mat = sheet_material && GET_DECL(sheet_material)
```

Comment:
Is this any different than calling `GET_DECL` directly? Doesn't it have a ternary that checks `ispath(foo)`? `foo && ispath(foo)` will always equal `ispath(foo)`, so this check seems redundant.